blocks/rss-alexa-feature-block/features/rss/xml.test.js

```diff
@@ -0,0 +1,148 @@
+// eslint-disable-next-line no-unused-vars
+import Consumer from 'fusion:consumer'
+import { Rss } from './xml'
+
+const articles = {
+  content_elements: [
+    {
+      display_date: '2020-04-07T15:02:08.918Z',
+      website_url: '/food/2020/04/07/tips-for-safe-hand-washing',
+      promo_items: {
+        basic: {
+          title: 'Hand Washing',
+          url: 'https://arc-anglerfish-arc2-prod-demo.s3.amazonaws.com/public/JTWX7EUOLJE4FCHYGN2COQAERY.png',
+          caption: 'Hand washing can be fun if you make it a song',
+          credits: { by: [{ name: 'Harold Hands' }] },
+        },
+      },
+      credits: {
+        by: [
+          { name: 'John Smith', _id: 'john-smith' },
+          { name: 'Jane Doe', _id: 'jane-doe' },
+        ],
+      },
+      headlines: { basic: 'Tips for Safe Hand washing' },
+      description: { basic: 'Tips to keep you wash for 20 seconds' },
+      subheadlines: { basic: 'This is from the subheadlines' },
+      content_elements: [
+        { type: 'text', content: 'try singing the happy birthday song' },
+        { type: 'text', content: 'be sure to wash your thumbs' },
+        {
+          _id: 'F3SWUDSWAFCCBIM6TKRM2J6DOA',
+          type: 'audio',
+          version: '0.10.6',
+          streams: [
+            {
+              url: 'https://clark.com/wp-content/uploads/2021/01/Ask-Clark_eBay-Truck-Scam_Jan-2021.mp3',
```

Comment:
You should use the example url from the ticket 'https://clark.com/wp-content/uploads/2021/01/Ask-Clark_GameStop_2021.mp3?_=4'